openpathsampling/dynamics_engine.py

```diff
@@ -296,6 +294,9 @@ def generate(self, snapshot, running=None):
             logger.info("Starting trajectory")
             log_freq = 10 # TODO: set this from a singleton class 
             while stop == False:
+                if self.options.get('n_frames_max', None) is not None :
```

Comment:
Do we need to use `get` here? It seems like either we need to include `n_frames_max` as in `default_options`, or we need to have a back-up in case the key isn't in the dictionary. But we don't need both, do we?